release/push-to-registry-cli.yaml

```diff
@@ -0,0 +1,218 @@
+# Copyright The Enterprise Contract Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: push-to-registry-cli
+spec:
+  description: Tekton pipeline to release Snapshots to a registry.
+  finally:
+  - name: cleanup
+    params:
+    - name: subdirectory
+      value: $(context.pipelineRun.uid)
+    - name: delay
+      value: "0"
+    taskRef:
+      params:
+      - name: url
+        value: https://github.com/konflux-ci/release-service-catalog.git
+      - name: revision
+        value: production
+      - name: pathInRepo
+        value: tasks/cleanup-workspace/cleanup-workspace.yaml
+      resolver: git
+    workspaces:
+    - name: input
+      workspace: release-workspace
+  params:
+  - description: The namespaced name (namespace/name) of the Release custom resource
+      initiating this pipeline execution
+    name: release
+    type: string
+  - description: The namespaced name (namespace/name) of the releasePlan
+    name: releasePlan
+    type: string
+  - description: The namespaced name (namespace/name) of the snapshot
+    name: snapshot
+    type: string
+  - default: github.com/enterprise-contract/ec-cli//policies/cli
+    description: JSON representation of the EnterpriseContractPolicy
+    name: enterpriseContractPolicy
+    type: string
+  - default: pipeline_intention=release
+    description: |
+      Extra rule data to be merged into the policy specified in params.enterpriseContractPolicy. Use syntax
+      "key1=value1,key2=value2..."
+    name: enterpriseContractExtraRuleData
+    type: string
+  - default: 40m0s
+    description: Timeout setting for `ec validate`
+    name: enterpriseContractTimeout
+    type: string
+  tasks:
+  - name: verify-access-to-resources
+    params:
+    - name: release
+      value: $(params.release)
+    - name: releasePlan
+      value: $(params.releasePlan)
+    - name: snapshot
+      value: $(params.snapshot)
+    - name: requireInternalServices
+      value: "false"
+    taskRef:
+      params:
+      - name: url
+        value: https://github.com/lcarva/release-service-bundles.git
```

Comment:
Perhaps add a comment and point to https://github.com/konflux-ci/release-service-catalog/pull/672?

Reply:
This yaml is generated by Kustomize so no comments here, but there are comments in the files under `release/src/base/tasks`.